### DISCUSSION TITLE
Newbie questions - what cross-seed/znab do?

### DISCUSSION THREAD

**Discussion by slrslr** [cross-seed/cross-seed#604] [Q&A] [answered]
I had a feedback to insufficient README and documentation, but it was closed and ignored, so I am asking here some newbie questions that to me fails to be sufficiently and clearly explained at:
https://github.com/cross-seed/cross-seed
https://www.cross-seed.org/docs/basics/getting-started

1. it does not mention which data "torznab" is supplying, ideally showing an example, considering that the reader does not know what torznab is and does not know any apps that produce it.

2. no examples of usage, how user can benefit from the app. For example: Can user supply the directory of .torrent files and the app will recursively examine all .torrent files automatically and find torrents with same payload (what about renamed files) on which number of trackers, and download that torrents as a .torrent files?

3. no link to a simple layman description on how to get a torznab link and the amount of trackers it can search.

The documentation is made stupidly, it speaks about installation instead of first explaining what the software can exactly achieve, in layman terms. The Github readme description in this regard is insufficient.

**Comment by zakkarry** [comment, marked as answer]
All of these questions were answered for you despite your poor attitude, and the documentation provides these answers as well. The README is not documentation and it links to the actual documentation. Your questions were not ignored. You have just refused to do anything beyond complain without merit.

1. Read what is told and provided to you in #603, which you didn't link to in your OP despite stating you were ignored and evidence to the contrary. I gave you screenshots of the exact spot where it links to apps to "produce" torznab and i explicitly told you what apps in my response, I told you if you needed to find out more you could search google for the specs for the protocol. We did not create the protocol and it is documented elsewhere. Once again our goal is not to be Wikipedia and make copies of information made available elsewhere.

2. https://www.cross-seed.org/docs/basics/getting-started#introduction - it says how you can benefit in the first paragraph of the first page you would see if you read the documentation. There are numerous other pages that you should read, you keep linking to getting-started. This is not a detailed account of every bit of functionality, that's why other pages on the documentation exist. Everything you have asked is answered.

3. I showed you the exact place where it tells you this in the documentation.
![image](https://github.com/cross-seed/cross-seed/assets/123845855/b32c7357-2c39-4145-a32d-8b4d142850c0)

I don't know what more you could possibly want.

Edit: There is also no need to send me direct emails crying about things I have shown you or that are answered already. That's bad form.

**Reply by slrslr** [reply]
Wrong again in numerous things. 

> I gave you screenshots of the exact spot where it links to apps to "produce" torznab

Thanks, but I do not care much. My aim was to point about insufficient documentation which is not useful for someone who do not know the apps you have mentioned and is not useful for other reasons i have mentioned here/in the issue.

> https://www.cross-seed.org/docs/basics/getting-started#introduction - it says how you can benefit

wrong again, read my issue to hopefully understand what it is missing, but it is OK, it is your choice if you do not want to provide additional information or at least links and instead waste more time of all readers.

> I showed you the exact place

Yes, but as i have said, you have failed to provide it (or link it) for everyone inside README or docs, possibly on the [getting-started#introduction](https://www.cross-seed.org/docs/basics/getting-started#introduction) where is the first occurrence of the "torznab".

And i have not cried (maybe you do). I have addressed your egoist post the way i could while you forbidden me to respond directly.

**Reply by zakkarry** [reply]
You do not care but found it necessary to make multiple posts about the same thing, and email me directly crying further about how victimized you were.

Hilarious.

It's covered in the documentation, everything you said has already been addressed. You seem to have the inability to read past the landing page, and that's not how documentation works. Read the entirety of the pages. If you are lazy, this is not going to be the project/software for you.

I've given you numerous links to where this is covered, the fact that you are unable to proceed past getting-started to understand software is not the failure of the documentation.

It only speaks to your ego and not mine that you found it necessary to try and have the "last word".

**Reply by slrslr** [reply]
Yes, you continue not to use your brain to trying to understand what i have said and instead repeating like a monkey and joking of me, you have proven what i have said about you. Not only ignorant and also rude. I guess you do not want my explanation, so i will not try to write it and clarify for the third time

**Reply by zakkarry** [reply]
whatever you say.
